flake.nix

```diff
@@ -45,6 +45,8 @@
 
     holochain.follows = "empty";
     lair.follows = "empty";
+    # lair repo does not have a flake
+    lair.flake = false;
```

Comment:
i think we can apply this to all "empty" inputs

Reply:
ah, so it's not because the repo that it gets set to doesn't have a flake, but the "empty" repo doesn't have a flake.

Reply:
the "empty" flake that is followed here and defined above is indeed a flake. but if you override any of these inputs with a url that does not point to a flake then you'll get an error as nix tries to look for a `flake.nix` at the content of the url.